### ISSUE TITLE
Shift+tab from message input doesn't work properly with screen readers

### ISSUE THREAD

**Issue by Neurrone** [UnigramDev/Unigram#1236]
Hi,

I'm using the NVDA screen reader with Unigram and noticed some bugs with how the shift+tab key works when navigating from the message input box, which automatically gains focus when I press enter on a conversation.

When the message is scrolled to the bottom (i.e, showing the most recent messages):

1. Pressing shift+tab once from message input speaks "list, link". The link is missing a label to indicate what it does.
2. Pressing shift+tab a second time speaks "link, <author>" which is fine.
3. Pressing shift+tab a third time actually brings the message list item in focus. The problem is that pressing shift+tab from message input when opening a conversation with unread messages often does not bring me to the first unread message, and seems to land me in random earlier messages. This makes it more difficult to quickly read only unread ones.

**Comment by Neurrone** [comment]
Bump?

**Comment by FrayxRulez** [comment]
Hey, sorry for not having replied to this ticket before.
The problem here is that it doesn't seem to be possible to focus a specific message (ie the last one) programmatically: I've looking into this a lot in the past, and there's no way to override this behavior.

**Comment by FrayxRulez** [comment]
This should be also a duplicate of #1183
